users.json

```diff
@@ -390,6 +330,40 @@
         "googleapis/google-cloud-cpp",
         "googleapis/google-cloud-cpp-spanner"
       ]
+    },
+    {
+      "team": "yoshi-contractors",
+      "users": [
+        "abhinav-qlogic",
```

Comment:
@danoscarmike this list should include all of the contractors. Are any missing to your knowledge?

Reply:
Added one additional user